src/null-safety/faq.md

```diff
@@ -172,13 +172,16 @@ The assert will be unnecessary when everything is fully migrated, but for now it
 
 ## How should I migrate a runtime null check that now shows as unnecessary?
 
-An explicit runtime null check, for example `if (arg == null) throw
-ArgumentError(...)`, will be flagged as an unnecessary comparison if you make
-`arg` non-nullable.
+The compiler flags an explicit runtime null check as an unnecessary
+comparison if you make `arg` non-nullable.
 
-But, the check *is* still needed if the program is a mixed-version one.  Until
-everything is fully migrated and the code switches to running with sound null
-safety, it will be possible for `arg` to be null.
+```dart
+if (arg == null) throw ArgumentError(...)`
+```
+
+You must include this check if the program is a mixed-version one.
+Until everything is fully migrated and the code switches to running
+with sound null safety, `arg` might be set to `null`.
```

Comment:
This is unrelated to js compilation, not sure if this was committed by accident

Reply:
This was to clean up the language and format around this check. It's the same content as what was there originally.